### PR TITLE
Update images

### PR DESCRIPTION
**Pull request by AnaDiaz85**

### CODE REVIEW

File: app/content/carto3-workspace/connections/carto-data-warehouse.md [hunk 1]
```diff
@@ -4,14 +4,16 @@ Every CARTO organization comes with an active data warehouse connection created
...
 ### Spatial Extension
 
-The CARTO Spatial Extension is enabled by default on the `carto_dw` connection. Check [this reference](https://docs.carto.com/spatial-extension-bq/sql-reference/overview/) to learn about the functions available in the extension.
+The CARTO Spatial Extension is enabled by default on the `CARTO Data Warehouse` connection. Check [this reference](https://docs.carto.com/spatial-extension-bq/sql-reference/overview/) to learn about the functions available in the extension.
 
 ### CARTO Demo Data
 
-The `carto_dw` connection grants access to a set of demo tables and tilesets that can be used to start creating maps and exploring the tools from the very beginning. They're available and ready to be used in your Data Explorer.
\ No newline at end of file
+The `CARTO Data Warehouse` connection grants access to a set of demo tables and tilesets that can be used to start creating maps and exploring the tools from the very beginning. They're available and ready to be used in your account's Data Explorer.
```
Comment (srtena) [inline app/content/carto3-workspace/connections/carto-data-warehouse.md:19]:
```suggestion
The `CARTO Data Warehouse` connection grants access to a set of demo tables and tilesets that can be used to start creating maps and exploring the tools from the very beginning. They're available and ready to be used in your  Data Explorer.
```

File: app/content/carto3-workspace/settings/configuring-your-organization.md [hunk 2]
```diff
@@ -1,23 +1,11 @@
 ## Configuring your organization
 
-CARTO 3 Workspace includes several options for configuring your organization. From the *Settings* menu, you can rename your organization, manage who can join your organization, view the details of your organization plan, register existing apps, invite users to join your organization, and delete your organization.
-
-### Renaming your organization
```
Comment (srtena) [inline app/content/carto3-workspace/settings/configuring-your-organization.md:5]: Are we getting rid of this section by mistake or on purpose?

File: app/content/carto3-workspace/settings/your-subscription.md [hunk 3]
```diff
@@ -20,6 +20,6 @@ The Plan features included are:
 - Connections
 - Applications
 
-![Settings your subscription](/img/cloud-native-workspace/settings/settings_your_subscription.png)
+![Account Settings your subscription](/img/cloud-native-workspace/account-settings/settings_your_subscription.png)
```
Comment (srtena) [inline app/content/carto3-workspace/settings/your-subscription.md:23]:
```suggestion
![Account Settings your subscription](/img/cloud-native-workspace/settings/settings_your_subscription.png)
```

**Review by srtena — CHANGES_REQUESTED** [review]
@AnaDiaz85 Just review those 3 comments and all OK from my side! Huge effort 👏

File: app/content/carto3-workspace/settings/configuring-your-organization.md (hunk 2 above)
Comment (AnaDiaz85) [reply app/content/carto3-workspace/settings/configuring-your-organization.md:5]: on purpose ;)

**Review by srtena — APPROVED** [review]

**Pull request merged**